src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,62 @@
 package core.basesyntax;
 
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
```

Comment:
Use LocalDate instead of Date. The LocalDate class provides various utility methods to obtain a variety of information. Here is some background why you should use it primarily: https://www.baeldung.com/migrating-to-java-8-date-time-api